src/update.ts

```diff
@@ -4,34 +4,26 @@ import { ColorifyConstants } from './api/constants/Colors'
 import { Messages } from './lib/constants/Messages'
 
 import * as pkg from '../package.json'
+import { getDistTag, getSimpleVersion } from './modules/utils'
 
 export function updateNotify() {
-  const notifier = updateNotifier({ pkg, updateCheckInterval: 1000 * 60 * 60 * 1 })
-  if (notifier.update && notifier.update.latest !== pkg.version) {
-    const oldVersion = notifier.update.current
+  const ONE_HOUR = 1000 * 60 * 60 * 1
```

Comment:
_non blocking_
You can declare this variable outside the `updateNotify` function

Reply:
I thought about it, but any other functions will use it.